src/components/HotspotListItem.tsx

```diff
@@ -89,7 +92,7 @@ const HotspotListItem = ({
       <Pressable onPress={handlePress}>
```

Comment:
I believe clicking on this is now supposed to expand/contract and show recently seen beacons.

![3 4 0 2021-07-12 at 11 06 47 AM](https://user-images.githubusercontent.com/1760821/125320398-48283880-e301-11eb-92ff-8761ea89bfff.jpg)

Reply:
Yeah that was a stretch goal and we didnt have a good way to get the data. So we are going without it for now. I created an issue on the blockchain API so that we can figure out a better way to get that data. https://github.com/helium/blockchain-http/issues/284

Reply:
Ok. I wonder if we should remove the chevron and click action? It's just kind of odd because there's no back button.